apps/prairielearn/elements/pl-details/pl-details.mustache

```diff
@@ -0,0 +1,30 @@
+<div class="accordion mb-3" id="pl-details-{{uuid}}">
+  <div class="accordion-item">
+    <h2 class="accordion-header" id="pl-details-header-{{uuid}}">
+      <button 
+        {{#show}}class="accordion-button"{{/show}}
+        {{^show}}class="accordion-button collapsed"{{/show}}
```

Comment:
Maybe this instead? Ditto elsewhere.

```suggestion
        class="accordion-button {{^show}}collapsed{{/show}}"
```

Reply:
I assume this is a nit, I actually find mine a little easier to read since the mustache syntax isn't highlighted as a string.

Reply:
It is indeed a nit, though my suggestion is consistent with the rest of the elements in the codebase.

Reply:
Man, I'm really hoping to get some better linting and highlighting in these files. I'll make your changes, but definitely on my radar.

Reply:
Changed in [f81b596](https://github.com/PrairieLearn/PrairieLearn/pull/11929/commits/f81b59665f57416b3f4efda0af559efa039fcf2c).